modules/gapi/samples/face_detection.cpp

```diff
@@ -170,6 +169,14 @@ std::vector<Face> buildFaces(const cv::Mat& scores,
 
     const float* reg_data = regressions.ptr<float>();
 
+    auto out_side = std::max(h, w);
+    auto in_side = 2 * out_side + 11;
```

Comment:
Magic values (2, 11)

Reply:
https://github.com/openvinotoolkit/open_model_zoo/blob/master/demos/face_detection_mtcnn_demo/python/mtcnn_utils.py#L73

Reply:
The same as the original demo